index.js

```diff
@@ -10,3 +11,46 @@ app.use("/", express.static(__dirname + '/app'));
 app.get('/', function (req, res) {
 	res.sendFile('/app/index.html',{root: __dirname})
 });
+
+const mongo = require('mongodb').MongoClient
+const mongoUrl = Config.config.mongoUrl;
+const mongoDatabase = Config.config.mongoDatabase;
+const mongoCollection = Config.config.mongoCollection;
+const collectEventBusData = Config.config.collectEventBusData;
+
+var mongoClient;
+
+var options = {
+	keepAlive: 1, 
+	connectTimeoutMS: 30000,
+	useNewUrlParser: true
+  };
+
+if (mongoUrl) {
+	mongo.connect(mongoUrl, options, function(err, client) {
+		if (err) throw err;
+		mongoClient = client;
+	});
+}
+
+function insertToMongo(data) {
+	if (!mongoClient || !collectEventBusData) {
+		return;
+	}
+	const db = mongoClient.db(mongoDatabase);
+	const collection = db.collection(mongoCollection);
+	collection.insertOne(data, (err, result) => {
+		if (err) throw err;
```

Comment:
since this exception is thrown from a callback, I assume it won't crash the whole amazon-connect-demo application?

Reply:
yes